python/lsst/cp/pipe/utils.py

```diff
@@ -0,0 +1,61 @@
+#
+# LSST Data Management System
+#
+# Copyright 2008-2019  AURA/LSST.
```

Comment:
Should switch this to the new copyright form.  I always steal from the `geom` package as some good soul already updated all of those.